packages/components/src/custom-select/index.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from 'downshift';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Dashicon } from '../';
+
+const itemToString = ( item ) => item.name;
+// This is needed so that in Windows, where
```

Comment:
Just checked this on Windows and the behaviour is identical to Mac: on arrow down/up the menu opens. I don't see that as a major issue though. If we did want the arrow keys to work with the menu closed we'd probably have to make sure the options always render, and hide them with CSS, but then it's not clear that we could limit that behaviour to Windows 🤷‍♀

Reply:
the issue here is the returned `changes`. @epiqueras just return the `selectedItem` because if you also return the rest of the `changes` then one of them is `isOpen` that comes as true :D

Reply:
Fixed :smile: